Makefile

```diff
@@ -486,6 +489,10 @@ verify-modules: generate-modules  ## Verify go modules are up to date
 		git diff; \
 		echo "go module files are out of date"; exit 1; \
 	fi
+	@if !(git diff --quiet HEAD -- go.sum go.mod $(PACKAGING_DIR)/go.mod $(PACKAGING_DIR)/go.sum); then \
```

Comment:
we don't have to diff top-level go.mod + go.sum again. let's just add the new ones to l.488